db/migrate/20210128134938_add_gias_fields_to_schools.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class AddGiasFieldsToSchools < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
Have you tested what would happen if you would revert the migration?

Reply:
```
== 20210128134938 AddGiasFieldsToSchools: reverting ===========================
-- remove_column(:networks, :secondary_contact_email, :string, {})
   -> 0.0020s
-- remove_column(:networks, :group_uid, :string, {})
   -> 0.0008s
-- remove_column(:networks, :group_id, :string, {})
   -> 0.0008s
-- remove_column(:networks, :group_type_code, :string, {})
   -> 0.0013s
-- remove_column(:networks, :group_type, :string, {})
   -> 0.0009s
-- remove_column(:schools, :secondary_contact_email, :string, {})
   -> 0.0010s
-- remove_column(:schools, :school_status_name, :string, {})
   -> 0.0007s
-- remove_column(:schools, :school_status_code, :string, {})
   -> 0.0007s
-- remove_column(:schools, :school_website, :string, {})
   -> 0.0016s
-- remove_column(:schools, :school_phase_name, :string, {})
   -> 0.0007s
-- remove_column(:schools, :school_phase_type, :string, {})
   -> 0.0007s
-- remove_column(:schools, :previous_school_urn, :string, {})
   -> 0.0007s
-- remove_column(:schools, :ukprn, :string, {})
   -> 0.0007s
-- remove_column(:schools, :school_type_name, :string, {})
   -> 0.0007s
-- rename_column(:schools, :school_type_code, :school_type)
   -> 0.0035s
-- remove_reference(:schools, :local_authority, {:foreign_key=>true, :type=>:uuid})
   -> 0.0237s
-- remove_reference(:schools, :local_authority_district, {:foreign_key=>true, :type=>:uuid})
   -> 0.0046s
-- drop_table(:local_authorities, {:id=>:uuid})
   -> 0.0025s
-- drop_table(:local_authority_districts, {:id=>:uuid})
   -> 0.0014s
== 20210128134938 AddGiasFieldsToSchools: reverted (0.0516s) ==================
```

is what is happening locally after I loaded gias data into my database.

Reply:
Perfect